subscriptionform/urls.py

```diff
@@ -2,17 +2,21 @@
 from django.conf.urls import include, url
 from django.conf.urls.static import static
 from django.contrib import admin
-from home.views import index, add_participant
-from sports.views import ParticipantDetail, ParticipantList
+from home.views import index
+from sports.views import ParticipantCreate, ParticipantDelete, ParticipantDetail, ParticipantList, ParticipantUpdate
+
 
 urlpatterns = [
 
     url(r'^$', index, name='home'),
     url(r'^admin/', include(admin.site.urls)),
     url('^', include('django.contrib.auth.urls')),
-    url('^add_participant/', add_participant, name='add_participant'),
-    url('^participants/$', ParticipantList.as_view()),
+    url('^participants/$', ParticipantList.as_view(), name='participants'),
```

Comment:
je kan aparte urls.py maken oor participant en daar alle urls in zetten voor die module.